pkg/storage/etcd/testing/utils.go

```diff
@@ -220,6 +221,7 @@ func (m *EtcdTestServer) waitUntilUp() error {
 // Terminate will shutdown the running etcd server
 func (m *EtcdTestServer) Terminate(t *testing.T) {
 	m.Client = nil
+	m.clientTransport.CloseIdleConnections()
```

Comment:
IIUC, the whole purpose of this PR is to call this method here, right? (Without this line, this whole PR seems to be no-op).
And to be honest, I don't fully understand the purpose of it. Why is it necessary? Won't transport automatically close its idle connections on removal?
Any link to which I could look into to understand this change?